src/main/java/woowa/sw/baseball/StrikeBallInfo.java

```diff
@@ -0,0 +1,31 @@
+package woowa.sw.baseball;
+
+/**
+ * strike, ball 데이터 구조(VO)를 가진 class
+ * author : sw
+ */
+public class StrikeBallInfo {
+    private int strikeCnt;
```

Comment:
```suggestion
    private int strikeCount;
```
변수명을 지을 때 약어는 사용하지 않는게 좋습니다.